frontend/ui/data/keyboardlayouts/keypopup/ar_AA_popup.lua

```diff
@@ -0,0 +1,162 @@
+return {
+    alef = {
+        "ا",
+        north = "أ",
+        northeast = "إ",
+        northwest = "ء",
+        east = "آ",
+        west = "ﭐ",
+    },
+    ba = {
+        "ب",
+        north = "ݒ",
+    },
+    jeem = {
+        "ج",
+        north = "ج",
+    },
+    daal = {
+        "د",
+    },
+    h_aa = { -- الهاء
+        "ه",
+        label = "ﻪ",
+    },
+    waw = {
+        "و",
+    },
+    zay = {
+        "ز",
+        north = "ژ",
+    },
+    ha = { -- الحاء
+        "ح",
+    },
+    tah = {
+        "ط",
+    },
+    yaa = {
+        "ي",
+        north = "ئ",
+    },
+    kaf = {
+        "ك",
+        north = "گ",
+        northeast = "ک",
+    },
+    lam = {
+        "ل",
+        north = "ﻻ",
+        northeast = "ﻷ",
+        northwest = "ﻹ",
+        east = "ﻵ",
+    },
+    meem = {
+        "م",
+    },
+    nun = {
+        "ن",
+    },
+    seen = {
+        "س",
+    },
+    ayin = {
+        "ع",
+    },
+    fah = {
+        "ف",
+        north = "ڤ",
+        northeast = "ڥ",
+        east = "ڢ",
+    },
+    saad = {
+        "ص",
+    },
+    qaf = {
+        "ق",
+        north = "ڨ",
+    },
+    raa = {
+        "ر",
+    },
+    sheen = {
+        "ش",
+        north = "ڜ",
+    },
+    taa = {
+        "ت",
+    },
+    thaa = {
+        "ث",
+    },
+    thaal = {
+         "ذ",
+    },
+    dhad = {
+       "ض",
+    },
+    th_aa = {
+       "ظ",
+    },
+    ghayn = {
+       "غ",
+    },
+    khaa = {
+       "خ",
+    },
+    alefmaqsoura = {
+        "ى",
+    },
+    hamza = {
+        "ء",
+    },
+    wawhamza = {
+         "ؤ",
+    },
+    laa = {
+          "ﻻ",
+    },
+    taamarbouta = {
+          "ة",
+          label = "ﺔ",
+    },
+   diacritic_fat_ha = {
+          "َ",
+    },
+   diacritic_damma = {
+          "ُ",
+    },
+   diacritic_kasra = {
+          "ِ",
+    },
+   diacritic_sukoon = {
+          "ْ",
+    },
+   diacritic_shadda = {
+          "ّ",
+    },
+   diacritic_tanween_fath = {
+          "ً",
+    },
+   diacritic_tanween_damm = {
+          "ٌ",
+    },
+   diacritic_tanween_kasr = {
+          "ٍ",
+    },
+--  Diacritics (al-Harakat الحركات)
+   diacritics = {
```

Comment:
Last nitpick: have the comment indented as what follows it.

Reply:
@poire-z Done.